src/crowdsourced/CrowdsourcedSection.tsx

```diff
@@ -0,0 +1,138 @@
+import { WalkBox, WalkContainer } from "../VaxComponents";
+// import WalkModal from "./modal/WalkInModal";
+import { getDistanceKm } from "../utils/distance";
+import { Coords } from "../location-picker/LocationPicker";
+import { Instruction, useCrowdsourcedLocations } from "./CrowdsourcedData";
+import { useState } from "react";
+import { Spinner } from "baseui/spinner";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCar, faWalking } from "@fortawesome/free-solid-svg-icons";
+import CrowdsourcedModal from "./modal/CrowdsourcedModal";
+
+export interface Props {
+  coords: Coords;
+  radiusKm: number;
+}
+
+export function CrowdSourcedSection({ coords, radiusKm }: Props) {
+  const locations = useCrowdsourcedLocations(coords, radiusKm);
+
+  const [selectedLocationIndex, setSelectedLocation] = useState<number>();
+  const [currentView, setCurrentView] = useState(6);
+  const openModal = (locationIndex: number) => {
+    setSelectedLocation(locationIndex);
+  };
+
+  const clearSelectedLocation = () => setSelectedLocation(undefined);
+
+  const loadMore = () => {
+    setCurrentView((latest) => latest + 6);
+  };
+
+  return "error" in locations ||
+    ("ok" in locations && locations.ok.length === 0) ? null : (
+    <div>
+      <CrowdsourcedModal
+        clearSelectedLocation={clearSelectedLocation}
+        location={
+          "ok" in locations && selectedLocationIndex !== undefined
+            ? locations.ok[selectedLocationIndex]
+            : undefined
+        }
+      />
+      <h2 className="WalkSection">
+        Crowdsourced Vaccination Locations<strong> - Open Today</strong>
```

Comment:
Do we want 'crowdsourced' as the title? Maybe it's good to make it clear, but to plays devils advocate would it make more sense to the user that it's simply a location that doesn't support the bookmyvaccine booking system? 

To the user, it doesn't matter how we got the data, what matters is how they book and find slots.

Reply:
I agree but I haven't heard a better name yet

"Other Vaccination Centres" ?

I think we can keep the code calling it Crowdsourced but I agree we need to change that wording before we ship